src/memary/agent/base_agent.py

```diff
@@ -73,16 +74,12 @@ def __init__(
         database = "neo4j"
 
         # initialize APIs
-        # OpenAI API
-        self.model = "gpt-3.5-turbo"
-        self.openai_api_key = os.environ["OPENAI_API_KEY"]
-        self.model_endpoint = "https://api.openai.com/v1"
+        self.load_llm_model(llm_model_name)
         self.openai_mm_llm = OpenAIMultiModal(
             model="gpt-4-vision-preview",
```

Comment:
I think we need to make this disabled if there is no OPENAI_KEY provided because we added the llama model as the default model.

Reply:
Yep I agree, just working right now to figure out how LlaVa through ollama works to set that as default